care/facility/forms.py

```diff
@@ -23,3 +23,4 @@ class DoctorsCountCreationForm(ModelForm):
     class Meta:
         model = HospitalDoctors
         fields = ["area", "count"]
+        labels = {"area": "Enter emergency contact number of your hospital"}
```

Comment:
Area is not the phone number. Please change it as directed in issue

Reply:
my mistake   will correct it